FlowControlHomework/FlowControl.cs

```diff
@@ -407,23 +411,22 @@ public void Loops()
                     break;
                 }
             }
-            Assert.IsTrue(passedCounter >= 1);  // Don't change this line.
+            Assert.IsTrue(passedCounter <= 1);  // Don't change this line.
```

Comment:
This should remain `passedCounter >= 1`. It's verifying that the code execution hits the `passedCounter++` line at least once.

You need to go into the loop at least once, but you'll need to change (or add) something inside the loop so it ends.

Reply:
That was me, I told him to change it to demonstrate how changing that affects how many times you would need to allow the loop to process.

Reply:
Changing it requires addition thought and calculation of what the initial int value is, to ensure we don't loop more an once. 

He successfully demonstrated this section with the original greater than sign.

Reply:
It still needs one change. Currently it's an infinite loop, so the test just keeps running in VS, but it fails on GitHub. `counter == 10` is never true, so the loop never breaks.